anvil/anvil-annotations/src/main/java/com/duckduckgo/anvil/annotations/ContributesPluginPoint.kt

```diff
@@ -29,6 +29,8 @@ import kotlin.reflect.KClass
  * }
  * ```
  */
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.RUNTIME)
```

Comment:
Incidentally improve the annotations by marking them with the correct retention and target